src/google/protobuf/parse_context.h

```diff
@@ -160,23 +159,6 @@ class PROTOBUF_EXPORT EpsCopyInputStream {
    private:
     absl::optional<int> token_;
   };
-#else
-  class LimitToken {
-   public:
-    LimitToken() = default;
-    explicit LimitToken(int token) : token_(token) {}
-    LimitToken(LimitToken&&) = default;
-    LimitToken& operator=(LimitToken&&) = default;
-
-    LimitToken(const LimitToken&) = delete;
-    LimitToken& operator=(const LimitToken&) = delete;
-
-    int token() const { return token_; }
```

Comment:
and switch this to be `&&` qualified instead of `const`, and add poisoning here as well